proposals/76:vaults-auctions/vatDetails.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-env node */
+import { dbTool, getInstanceBoardId, HOME } from '@agoric/synthetic-chain';
+import { writeFile } from 'fs/promises';
+import dbOpenAmbient from 'better-sqlite3';
+
+const { env } = process;
+
+const swingstorePath = '~/.agoric/data/agoric/swingstore.sqlite';
+
+// XXX: imports of assert and NonNullish aren't working.
+export const assert = (cond, msg = 'check failed') => {
+  if (!cond) {
+    throw Error(msg);
+  }
+};
+/** @type {<T>(val: T | undefined) => T} */
+export const NonNullish = val => {
+  if (!val) throw Error('required');
+  return val;
+};
+
+export const lastAuctionInstancePathname = `${env.HOME}/.agoric/lastAuctioneerInstance.json`;
+
+export const recordAuctioneerInstance = async () => {
+  const oldAuctionInstance = await getInstanceBoardId('auctioneer');
+  assert(oldAuctionInstance, 'no auction instance found');
+  console.log('old auction instance ', oldAuctionInstance, env.HOME);
+
+  await writeFile(lastAuctionInstancePathname, oldAuctionInstance);
+};
+
+// duplicated from vat-status in synthetic-chain
```

Comment:
There's a dance here because getting changes into synthetic-chain waits on a release cycle. I've been adding tools to each a3p test, then moving them to synthetic-chain to be shared once they've proved their worth.

Here I'm adding a method to the swingStore object that gets returned. It's being added to synthetic-chain in https://github.com/Agoric/agoric-3-proposals/pull/174, but it's not there yet.